include/freerdp/client/rdpsnd.h

```diff
@@ -37,9 +37,7 @@ struct _RDPSND_WAVE
 	UINT16 wTimeStampA;
 	UINT16 wTimeStampB;
 
-	UINT16 wLatency;
```

Comment:
I don't know if people using this include have something to say about that but it looks like API + ABI break.